### PR TITLE
feat(no-access): add no access message

### PR DESCRIPTION
**Pull request by thanh-nguyen-dang**
Jira Ticket: [PXP-10908](https://ctds-planx.atlassian.net/browse/PXP-10908)

### New Features
Add no access description for File page

### Deployment changes

[PXP-10908]: https://ctds-planx.atlassian.net/browse/PXP-10908?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by ocshawn — COMMENTED** [review]
the language did not look right, please update

**Review by paulineribeyre — DISMISSED** [review]

**Review by ocshawn — APPROVED** [review]

File: src/CoreMetadata/CoreMetadataHeader.jsx [hunk 1]
```diff
@@ -95,7 +95,10 @@ class CoreMetadataHeader extends Component {
         } else {
           // set to empty so it wont check again
           this.setState({
-            downloadButton: (<React.Fragment />),
+            downloadButton: (
+              <React.Fragment>
+                <p className='h3-typo' id='no-permission'>You don&apos;t have permission to download this data</p>
```
Comment (paulineribeyre) [inline src/CoreMetadata/CoreMetadataHeader.jsx:100]:
```suggestion
                <p className='h3-typo' id='no-permission'>You do not have access to download this data.</p>
```

File: src/CoreMetadata/CoreMetadataHeader.jsx [hunk 2]
```diff
@@ -95,7 +95,10 @@ class CoreMetadataHeader extends Component {
         } else {
           // set to empty so it wont check again
           this.setState({
-            downloadButton: (<React.Fragment />),
+            downloadButton: (
+              <React.Fragment>
+                <p className='h3-typo' id='no-permission'>You do not have access to download this data.</p>
+              </React.Fragment>),
```
Comment (ocshawn) [inline src/CoreMetadata/CoreMetadataHeader.jsx:101]:
```suggestion
                <p className='h3-typo' id='no-permission'>You do not have access to download this data.</p>
              ),
```

**Review by paulineribeyre — APPROVED** [review]

**Comment by paulineribeyre** [comment]
It passed the indexd GUI test on run #4 and the core metadata tests on run #7

**Pull request merged**